armi/reactor/tests/test_components.py

```diff
@@ -285,12 +344,25 @@ def test_preserveMassDuringThermalExpansion(self):
             )
 
     def test_volumeAfterClearCache(self):
+        """
+        Test volume after cache has been cleared.
+
+        .. test:: Clear cache after a dimensions updated
+            :id: T_ARMI_COMP_VOL1
+            :tests: R_ARMI_COMP_VOL
```

Comment:
Can you also put test crumbs for this requirement on a couple of the `test_getVolume()` and `test_getArea()` tests in this file?

Thanks!